library/psa_crypto.c

```diff
@@ -1498,17 +1498,47 @@ static psa_status_t psa_validate_key_attributes(
     const psa_key_attributes_t *attributes,
     psa_se_drv_table_entry_t **p_drv )
 {
-    psa_status_t status;
+    psa_status_t status = PSA_ERROR_INVALID_ARGUMENT;
+    psa_key_lifetime_t lifetime = psa_get_key_lifetime( attributes );
 
-    if( attributes->core.lifetime != PSA_KEY_LIFETIME_VOLATILE )
+    /* Check there is a proper handler for this lifetime */
+    if ( PSA_KEY_LIFETIME_GET_LOCATION( lifetime )
```

Comment:
Is there any intent to support persistent SE-stored keys without having to require PSA Crypto Storage?

Reply:
No, because the metadata is always in internal storage. Due to the interface design (the application only remembers the key id, not its lifetime), it's difficult to have metadata spread over multiple storage areas.